blocks/init/src/Blocks/components/layout/layout.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Layout component - Grid Columns grid.
+ *
+ * @package EightshiftBoilerplate
+ */
+
+use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$layoutUse = Components::checkAttr('layoutUse', $attributes, $manifest);
+if (!$layoutUse) {
+	return;
+}
+
+$componentClass = $manifest['componentClass'] ?? '';
+$additionalClass = $attributes['additionalClass'] ?? '';
+$blockClass = $attributes['blockClass'] ?? '';
+$selectorClass = $attributes['selectorClass'] ?? $componentClass;
+
+$layoutItems = Components::checkAttr('layoutItems', $attributes, $manifest);
+$layoutTag = Components::checkAttr('layoutTag', $attributes, $manifest);
+$layoutType = Components::checkAttr('layoutType', $attributes, $manifest);
+$layoutLoadMoreId = Components::checkAttr('layoutLoadMoreId', $attributes, $manifest);
+
+$layoutClass = Components::classnames([
+	Components::selector($componentClass, $componentClass),
+	Components::selector($blockClass, $blockClass, $selectorClass),
+	Components::selector($additionalClass, $additionalClass),
+]);
+
+$unique = Components::getUnique();
+
+?>
+
+<<?php echo esc_attr($layoutTag); ?>
+	class="<?php echo esc_attr($layoutClass); ?>"
+	data-id="<?php echo esc_attr($unique); ?>"
+	data-layout-type="<?php echo esc_attr($layoutType); ?>"
+>
+	<?php echo Components::outputCssVariables($attributes, $manifest, $unique); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
You shouldn't need to add the `// phpcs:ignore` comment here, this is a safe method

https://github.com/infinum/eightshift-coding-standards/blob/main/Eightshift/Sniffs/Security/ComponentsEscapeSniff.php#L146-L147

Reply:
tnx. I have fixed it on all components